cluster/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -142,6 +142,12 @@ rules:
   - delete
   - get
   - update
+- apiGroups:
```

Comment:
Looks like we need the same change in the integration tests (ceph_manifests.go)

Reply:
Made the requested changes.